trulens/utils/colab.py

```diff
@@ -0,0 +1,60 @@
+# Some utilities to help with colab/jupyter notebook demos.
+
+import importlib
+from pathlib import Path
+import subprocess
+import sys
+
+
+def install_if_not_installed(packages):
```

Comment:
Are these used anywhere?

Reply:
In notebooks to be run easily in colab but I see none of them are presently in this PR. They are part of the visualizations PR. Exception to include them in this PR?